telegram/_passport/passportfile.py

```diff
@@ -137,8 +139,13 @@ def de_list_decrypted(
             return ()
 
         return tuple(
-            cls.de_json_decrypted(passport_file, bot, credentials[i])
-            for i, passport_file in enumerate(data)
+            filter(
```

Comment:
this will also remove `False` values, in addition to `None` values. Maybe the docstring should say "Filters out any falsy values"

Reply:
Changing to only filter out `None`